docs_user/modules/con_adoption-prerequisites.adoc

```diff
@@ -18,6 +18,15 @@ endif::[]
 ** link:{defaultOCPURL}/nodes/overview-of-nodes[Overview of nodes]
 ** link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors]
 ** link:{defaultOCPURL}/machine_configuration/index[Machine configuration overview]
+* Make sure to set the correct {rhos_acro} project namespace to execute commands in.
+ifeval::["{build_variant}" == "ospdo"]
+[NOTE]
+In Director Operator adoption the adopted namespace would be "openstack", the adopting namespace would be different i.e. "rhoso".
+endif::[]
+[source, shell]
+----
+$ oc project $NAMESPACE
```

Comment:
I would just use the literal example `oc project openstack` that majority of customers will likely want to use, instead of using `$NAMESPACE`.

Reply:
I think @bogdando asked for this to be parameterized no ? 
and for the ospdo version that would be the wrong namespace to use , so how to get around that ? an if condition?

Reply:
Yes either an if condition or just the `[NOTE]` that you already included?

Reply:
ok

Reply:
done